### PR TITLE
added the optional strokeWidth property in line series and in docs

### PR DESCRIPTION
**Pull request by jckr**
added the strokeWidth property to lineSeries, so not all lines have to be the same width

### CODE REVIEW

File: src/lib/plot/series/line-series.js [hunk 1]
```diff
@@ -100,7 +100,11 @@ class LineSeries extends AbstractSeries {
         ref="line"
         className="rv-xy-plot__series rv-xy-plot__series--line"
         transform={`translate(${marginLeft},${marginTop})`}
-        style={{opacity: 0, strokeDasharray: STROKE_STYLES[strokeStyle]}}/>
+        style={{
+          opacity: 0,
+          strokeDasharray: STROKE_STYLES[strokeStyle],
+          strokeWidth
```
Comment (bulyonov) [inline src/lib/plot/series/line-series.js]: Does it work undefined/null stroke widths?

File: docs/xy-plot.md [hunk 2]
```diff
@@ -49,21 +49,22 @@ XYPlot is a wrapper for series, hints, axes and other components. Most of these
 * `opacity` (optional)
 * `fill` (optional)
 * `stroke` (optional)
+* `strokeWidth` (optional), `strokeStyle` (optional) - to control the width of the line series and whether they are dashed or solid.
```
Comment (bulyonov) [inline docs/xy-plot.md]: @jckr, I suppose, you should put these changes separately: these properties don't support ranges, domains and types (as others do). I'd suggest to add them to `LineSeries`.

File: docs/xy-plot.md [hunk 3]
```diff
@@ -49,21 +49,22 @@ XYPlot is a wrapper for series, hints, axes and other components. Most of these
...
-| `BarSeries`      |  +  |  +  | +       |     +     |        |
-| `HeatmapSeries`  |  +  |  +  | +       |     +     |        |
+|                  | `x` | `y` | `color` | `opacity` | `size` | `strokeStyle` | `strokeWidth` |
+|------------------|-----|-----|---------|-----------|--------|-------------------------------|
+| `LineSeries`     |  +  |  +  | +       |           |        |       +       |       +       |
+| `AreaSeries`     |  +  |  +  | +       |           |        |               |               |
+| `LinemarkSeries` |  +  |  +  | +       |     +     | +      |       +       |       +       |
+| `MarkSeries`     |  +  |  +  | +       |     +     | +      |               |               |
+| `BarSeries`      |  +  |  +  | +       |     +     |        |               |               |
+| `HeatmapSeries`  |  +  |  +  | +       |     +     |        |               |               |
```
Comment (bulyonov) [inline docs/xy-plot.md]: @jckr , same here

File: src/lib/plot/series/line-series.js (hunk 1 above)
Comment (jckr) [inline src/lib/plot/series/line-series.js]: yes.

File: docs/xy-plot.md (hunk 2 above)
Comment (jckr) [inline docs/xy-plot.md]: ok.

**Pull request merged**